eos/form-factors/observables.cc

```diff
@@ -2149,6 +2150,65 @@ namespace eos
         return ObservableGroup(imp);
     }
 
+    // 0 -> PP
+    // {{{
+
+    // 0 -> pi pi
+    // {{{
+    ObservableGroup
+    make_vacuum_to_pipi_form_factors_group()
+    {
+        auto imp = new Implementation<ObservableGroup>{
+            R"(Form factors for $0 \to \pi \pi$ transitions)",
+            R"(Pseudo observables representing the full basis of $0 \to \pi \pi$ form factors. )"
+            R"(The specific parametrization can be chosen via the "form-factors" option.)",
+            {
+                make_form_factor_adapter("0->pipi::Abs{f_+}^2(q2)", R"(|f_+^{0\to \pi\pi}(q^2)|^2)",
+                        &FormFactors<VacuumToPP>::abs2_f_p, std::make_tuple("q2")),
+
+                make_form_factor_adapter("0->pipi::Arg{f_+}(q2)", R"(\textrm{arg}(f_+^{0\to\pi\pi}(q^2)))",
+                        &FormFactors<VacuumToPP>::arg_f_p, std::make_tuple("q2")),
+
+                make_form_factor_adapter("0->pipi::Re{f_+}(Re{q2},Im{q2})", R"(\textrm{Re}(f_+^{0\to\pi\pi}(q^2)))",
+                        &FormFactors<VacuumToPP>::re_f_p, std::make_tuple("Re{q2}", "Im{q2}")),
+
+                make_form_factor_adapter("0->pipi::Im{f_+}(Re{q2},Im{q2})", R"(\textrm{Im}(f_+^{0\to\pi\pi}(q^2)))",
+                        &FormFactors<VacuumToPP>::im_f_p, std::make_tuple("Re{q2}", "Im{q2}")),
+
+                make_observable("0->pipi::b_0", R"(b_0^{0 \to \pi\pi})", Unit::None(),
```

Comment:
This name should have a suffix, indicating that it is specific to ``KKRvD2024``.